### PR TITLE
Add functions for seeing ounces consumed by user and testing of those…

### PR DESCRIPTION
**Pull request by asutherland91**
I wrote two functions for checking users daily ounces consumed and also weekly ounces consumed and the testing for both of those functions. I also added in the days.js library for use into my functions. I left space in my code for easier merging due to an inevitable merge conflict.

### CODE REVIEW

**Comment by asutherland91** [comment]
Merge conflict resolved!

File: src/UserRepository.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class UserRepository {
     constructor(users) {
         this.users = users;    
-   
+    }
```
Comment (hjawad22) [inline src/UserRepository.js:4]: Thanks for adding this, got away from us last time around!

File: src/classes/Hydration.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
+const dayjs = require('dayjs')
+
 class Hydration {
   constructor(hydrationData) {
```
Comment (hjawad22) [inline src/classes/Hydration.js:4]: Great work on bringing in that hydration data!

**Comment by hjawad22** [comment]
Conflict resolved!

**Pull request merged**